recipes/logr/0.1.0/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class LogrConan(ConanFile):
+    name = "logr"
+    license = "BSD 3-Clause License"
+    homepage = "https://github.com/ngrodzitski/logr"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Logger frontend substitution for spdlog, glog, etc for server/desktop applications"
+    topics = ("logger", "development", "util", "utils")
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["CMakeLists.txt"]
+
+    options = { "backend": ["spdlog", "glog", "log4cplus", "log4cplus-unicode", None] }
+    default_options = { "backend": "spdlog"}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires("fmt/7.1.2")
+
+        if self.options.backend == "spdlog":
+            self.requires("spdlog/1.8.0")
+        elif self.options.backend == "glog":
+            self.requires("glog/0.4.0")
+        elif self.options.backend == "log4cplus":
+            self.requires("log4cplus/2.0.5")
+            self.build_requires_options["log4cplus"].unicode = False
+        elif self.options.backend == "log4cplus-unicode":
+            self.requires("log4cplus/2.0.5")
+            self.build_requires_options["log4cplus"].unicode = True
```

Comment:
This does not seem to change anything for logr https://github.com/conan-io/conan-center-index/pull/3812/files#diff-ad137baee4c6c3f95ccebfa8aa275fa0c0d5fff4e1f37e8b7ca65d2541479aeaR73

would it be possible to just have one and let the consumer set his own options?